progression/src/main/java/me/mykindos/betterpvp/progression/tree/fishing/model/FishingLoot.java

```diff
@@ -14,6 +15,6 @@ public interface FishingLoot {
      * Called whenever this loot is caught
      * @param event The event
      */
-    void processCatch(PlayerFishEvent event);
+    ItemStack processCatch(PlayerFishEvent event);
```

Comment:
Some of this can be solved by checking for it, but the main issue is that I was having difficulty getting the ItemStack the player received. What is a better way to solve this problem?

Reply:
Solved by [doing below](https://github.com/Mykindos/BetterPvP/pull/124#discussion_r1362992878)